api/src/opentrons/protocol_runner/legacy_context_plugin.py

```diff
@@ -129,7 +129,7 @@ def _handle_legacy_command(self, command: LegacyCommand) -> None:
         pe_actions = self._legacy_command_mapper.map_command(command=command)
         self._actions_to_dispatch.put(pe_actions)
 
-    def _handle_equipment_loaded(self, load_info: LegacyLoadInfo) -> None:
+    def _handle_equipment_loaded(self, load_info: LoadInfo) -> None:
         """Handle an equipment load reported by the APIv2 protocol.
```

Comment:
Maybe we can update these docstrings to be clearer too.
```suggestion
        """Handle an equipment load reported by the legacy APIv2 protocol.
```